pkg/controller/resource/resource.go

```diff
@@ -26,10 +27,25 @@ import (
 
 // Annotations on app to enable chaos on it
 const (
-	ChaosAnnotationKey   = "litmuschaos.io/chaos"
 	ChaosAnnotationValue = "true"
 )
 
+var (
+	ChaosAnnotationKey = getCustomAnnotation()
+)
+
+func getCustomAnnotation() string {
+	//ChaosAnnotationKey := "litmuschaos.io/chaos"
+
+	annotationKey := os.Getenv("CUSTOM_ANNOTATION")
```

Comment:
use os.LookupEnv

Reply:
No need @chandankumar4 , willl handle the empty string case.